src/main/java/org/mockito/internal/configuration/DefaultDoNotMockEnforcer.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2019 Mockito contributors
+ * This program is made available under the terms of the MIT License.
+ */
+package org.mockito.internal.configuration;
+
+import java.lang.annotation.Annotation;
+
+import org.mockito.DoNotMock;
+import org.mockito.plugins.DoNotMockEnforcer;
+
+public class DefaultDoNotMockEnforcer implements DoNotMockEnforcer {
+
+    @Override
+    public String checkTypeForDoNotMockViolation(Class<?> type) {
+        for (Annotation annotation : type.getAnnotations()) {
+            if (annotation.annotationType().getName().endsWith("org.mockito.DoNotMock")) {
+                String exceptionMessage =
+                    type + " is annotated with org.mockito.@DoNoMock and can't be mocked.";
```

Comment:
Shouldn't it be `@org.mockito.DoNotMock` ? Maybe adding the actual annotation might be useful as well.

Reply:
I think this is the official notation, but I have changed it to yours as that is probably more intuitive.